packages/mattermost-redux/src/reducers/requests/channels.ts

```diff
@@ -59,10 +60,21 @@ function getAllChannels(state: RequestStatusType = initialRequestState(), action
     );
 }
 
+function getChannelsMembersCategories(state: RequestStatusType = initialRequestState(), action: GenericAction): RequestStatusType {
+    return handleRequest(
+        ChannelTypes.CHANNELS_MEMBERS_CATEGORIES_REQUEST,
+        ChannelTypes.CHANNELS_MEMBERS_CATEGORIES_SUCCESS,
+        ChannelTypes.CHANNELS_MEMBERS_CATEGORIES_FAILURE,
+        state,
+        action,
+    );
+}
+
 export default (combineReducers({
     getChannels,
     getAllChannels,
     myChannels,
     createChannel,
     updateChannel,
+    getChannelsMembersCategories,
```

Comment:
new request added which tracks the loading of channel, channels member and categories. this was done to keep the logic in actions instead of being in the component

Reply:
Do we want to add this for the new code path? I know we were just discussing bringing back the campaign for removing these. 0/5 though since it could be more convenient to track request state here for this component